corpus/structure/CSIPSTR2/testCase.xsd

```diff
@@ -0,0 +1,1746 @@
+
```

Comment:
This should be the test case XSD not the HTML page from GitHub

Reply:
Done